packages/core-mobile/app/services/fcm/AppCheckService.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  firebase,
+  FirebaseAppCheckTypes
+} from '@react-native-firebase/app-check'
+import Logger from 'utils/Logger'
+
+class AppCheckService {
+  init = (): void => {
+    const rnfbProvider = firebase
+      .appCheck()
+      .newReactNativeFirebaseAppCheckProvider()
+    rnfbProvider.configure({
+      android: {
+        provider: __DEV__ ? 'debug' : 'playIntegrity',
+        debugToken:
+          'some token you have configured for your project firebase web console'
```

Comment:
how can one use this? do we just manually paste it in? should we put it in an env?

Reply:
there's env, have it on my todo list to put it in cloud

Reply:
updated in [Add Debug token](https://github.com/ava-labs/core-mobile/pull/1659/commits/d97b505d052cc7592e5f4434a6517f5aecd99e60)
Same token, it's trivial to create new one trough firebase console but really not too important